sheet_test.go

```diff
@@ -463,3 +465,30 @@ func TestAttrValToFloat(t *testing.T) {
 	assert.NoError(t, err)
 	assert.Equal(t, 42.1, got)
 }
+
+func TestSetSheetBackgroundFromBytes(t *testing.T) {
+	files := map[string]string{
+		"excelize.svg":          ".svg",
+		"test/images/excel.emf": ".emf",
+		"test/images/excel.emz": ".emz",
+		"test/images/excel.gif": ".gif",
+		"test/images/excel.jpg": ".jpg",
+		"test/images/excel.png": ".png",
+		"test/images/excel.tif": ".tif",
+		"test/images/excel.wmf": ".wmf",
+		"test/images/excel.wmz": ".wmz",
+	}
+	f := NewFile()
+	index := f.NewSheet("sheet1")
```

Comment:
The workbook has a default worksheet named "Sheet1" by default when it has been created, so we needn't create that.

In other ways, could we test it by set worksheet background images for multi worksheets instead of creating many workbooks?